### DISCUSSION TITLE
Telescope :projects command in alpha-nvim doc not working

### DISCUSSION THREAD

**Discussion by TikiCat7** [LazyVim/LazyVim#1175] [Q&A]
I'm trying to get the telescope projects (view recent working directories) to work from the nvim start screen. I'm following the alpha-nvim documents here: https://www.lazyvim.org/extras/util/project#alpha-nvim. I've added it that snippet to my plugins sectionm however getting this error when running the command from start.

<img width="620" alt="Screenshot 2023-07-21 at 11 39 23 AM" src="https://github.com/LazyVim/LazyVim/assets/15680763/61714e31-36d0-4662-9f5d-fb7e603dc971">

Are there additional configurations for telescope I'm missing? I am just using the default lazyvim with just a few plugins modified.

**Comment by folke** [comment]
You do not need to add those setting manually. All you have to do is include the projects extra as is mentioned on the top of that doc. https://www.lazyvim.org/extras/util/project
